python_modules/dagster/dagster/_core/storage/partition_status_cache.py

```diff
@@ -38,6 +39,14 @@
 )
 
 
+class PartitionStatus(Enum):
```

Comment:
I would call this `AssetPartitionStatus`, because e.g. MATERIALIZED only makes sense in the context of an asset.